www/templates/diary/detail.html

```diff
@@ -67,15 +62,24 @@
                 ng-click="refreshMap()">Refresh</button>
             <button class="button button-stable button-block"
                 ng-click="getIndividualSuggestion()">Get Suggestion</button>
+<<<<<<< HEAD
             <button class="button button-stable button-block"
                 ng-click="clickReview()">Get Reviews</button>
+=======
```

Comment:
You have an unresolved merge conflict here. I am surprised that this worked. Did you test after resolving the merge conflict?